### PR TITLE
Update to the Catalan Lenguage

### PR DESCRIPTION
**Pull request by Minepika55**
### Description of Changes
Updated the Catalan lenguage to be more accurate and to add new translations.

### Rationale behind Changes
The lenguage had not been updated in years and had some errors and many parts without translation. I mainly tried to focus on translating untranslated error messages.

### Suggested Testing Steps
Probably try to see if some translations are too long to fit inside some menus.

### CODE REVIEW

**Comment by Minepika55** [comment]
Sorry about the two other commits with only one change, still pretty new to GitHub and i thought i had to make a change any time i wanted to merge commits if i failed the first time.

File: locales/ca_ES/pcsx2_Main.po [hunk 1]
```diff
@@ -34,7 +34,7 @@ msgstr "Error d'anàlisi"
 #: common/include/Utilities/Exceptions.h:279
 msgid "Your machine's hardware is incapable of running PCSX2.  Sorry dood."
 msgstr ""
-"El hardware del teu ordinador no es capaç de fer servir PCSX2. Ho sentim "
+"El hardware del teu ordinador no es capaç de fer servir PCSX2. Ho sentim. "
```
Comment (refractionpcsx2) [inline locales/ca_ES/pcsx2_Main.po:37]: Any reason this isn't inside the first quotes?

**Comment by refractionpcsx2** [comment]
As I said to the nice person updating the French translation, I appreciate the updates, however we're about to scrap the WX GUI, which means all the current translations will disappear, since the strings for the new UI are going to be mostly different.  So I will accept the PR but just be aware that it may not be around for too long.

**Comment by Minepika55** [comment]
> As I said to the nice person updating the French translation, I appreciate the updates, however we're about to scrap the WX GUI, which means all the current translations will disappear, since the strings for the new UI are going to be mostly different. So I will accept the PR but just be aware that it may not be around for too long.

Oh, i see. Well in that case at least we have the basis of the translation for when the UI changes. I would love to try and translate  the new UI to catalan once it's done.

**Comment by refractionpcsx2** [comment]
Due to the looming changeover to Qt this is going to become irrelevant.  Once we are ready for translations in Qt then we will be happy to receive a translation for that, but for now this is kind of obsolete.

Thanks.

**Pull request closed without merging**